src/graphql/resolvers/queries/workshop.js

```diff
@@ -17,5 +19,13 @@ export const fieldResolvers = {
       if (!parent.tags) return [];
       return parent.tags;
     },
+    favoritedBy: async ({ id }, __, { dataSources: { firestore } }) =>
+      favoritedByFunc(id, firestore),
+
+    favoriteCount: async ({ id }, __, { dataSources: { firestore } }) => {
```

Comment:
You can drop the async.. you're now awaiting but returning a promise and that's all that's needed.